resources/functionMap.php

```diff
@@ -3912,14 +3912,14 @@
 'HaruPage::textOut' => ['bool', 'x'=>'float', 'y'=>'float', 'text'=>'string'],
 'HaruPage::textRect' => ['bool', 'left'=>'float', 'top'=>'float', 'right'=>'float', 'bottom'=>'float', 'text'=>'string', 'align='=>'int'],
 'hash' => ['non-empty-string|false', 'algo'=>'string', 'data'=>'string', 'raw_output='=>'bool'],
-'hash_algos' => ['list<non-empty-string>'],
+'hash_algos' => ['non-empty-list<non-falsy-string>'],
 'hash_copy' => ['HashContext', 'context'=>'HashContext'],
 'hash_equals' => ['bool', 'known_string'=>'string', 'user_string'=>'string'],
 'hash_file' => ['non-empty-string|false', 'algo'=>'string', 'filename'=>'string', 'raw_output='=>'bool'],
 'hash_final' => ['non-empty-string', 'context'=>'HashContext', 'raw_output='=>'bool'],
 'hash_hkdf' => ['non-empty-string|false', 'algo'=>'string', 'key'=>'string', 'length='=>'int', 'info='=>'string', 'salt='=>'string'],
 'hash_hmac' => ['non-empty-string|false', 'algo'=>'string', 'data'=>'string', 'key'=>'string', 'raw_output='=>'bool'],
-'hash_hmac_algos' => ['list<non-empty-string>'],
+'hash_hmac_algos' => ['non-empty-list<non-falsy-string>'],
```

Comment:
This function always returns non-empty-list and neither `'0'` nor `''`. 
https://www.php.net/manual/en/function.hash-hmac-algos.php